src/cs-scene/CelestialAnchor.hpp

```diff
@@ -79,7 +79,7 @@ class CS_SCENE_EXPORT CelestialAnchor {
   virtual void   setAnchorScale(double dScale);
 
   /// Called regularly by the Universe if registered.
-  virtual void update(double tTime, CelestialObserver const& oObs) {
+  virtual void update(double, CelestialObserver const&) {
```

Comment:
I don't like this solution. I split the declaration and assignment to avoid this warning. Also since this is a `/W4` warning we could in theory ignore it completely in the case we want to stay at `/W3`.